Packet++/header/HttpLayer.h

```diff
@@ -15,6 +15,17 @@
 #endif
 #endif
 
+#ifndef PCPP_DEPRECATED_MSG
```

Comment:
please remove this.

Reply:
Please see comment https://github.com/seladb/PcapPlusPlus/pull/1289#discussion_r1508709570